### DISCUSSION TITLE
Animate Mantine Modal height using framer-motion or any other library

### DISCUSSION THREAD

**Discussion by SarkarKurdish** [mantinedev/mantine#3267] [Q&A]
Is there is any API to animate mantine modal height?

**Comment by rtivital** [comment]
No, it is not supported

**Reply by SarkarKurdish** [reply]
Thanks I've used a workaround

**Comment by SarkarKurdish** [comment]
I've found a workaround which works really great for my case

Hide the header of the mantine modal and wrap all the content in a framer motion div and animate the div which works great

```
    <Modal
      closeOnClickOutside={false}
      withCloseButton={false}
      centered
    >
      <motion.div
        animate={{ height: "auto" }}
        transition={{ duration: 0.3 }}
      >
      </motion.div>
    </Modal>
```
